internal/test/framework/framework.go

```diff
@@ -91,12 +91,29 @@ func (f *TestFramework) getKubeClient() error {
 func (f *TestFramework) createWindowsVM() error {
 	// TODO: Add validation for the environment variables to see if they're properly populated or not
 	kubeconfig := os.Getenv("KUBECONFIG")
+	if kubeconfig == "" {
```

Comment:
I had a TODO here - https://github.com/openshift/windows-machine-config-operator/pull/113/commits/116103845216fb648b9ce342483dc6878952de4a#diff-5465ccc52b3a24461a98f029910cf60aR92, thanks for working on this @sebsoto

Reply:
Forgot to take it out, updated with it removed